keps/sig-testing/2420-reducing-kubernetes-build-maintenance/README.md

```diff
@@ -556,6 +556,7 @@ Major milestones might include:
 - 2020-02-08 - KEP Implementable [#2469](https://github.com/kubernetes/enhancements/pull/2469)
 - 2020-04-01 - KEP Alpha, Beta in Kubernetes 1.21
   - There is no distinct alpha/beta for this KEP, only alpha/beta (implemented at HEAD) vs stable (all supported branches)
+- 2020-08-16 - Retroactive implemented declaration
```

Comment:
Aren't all implemented declarations retroactive? /shrug

Reply:
I consider it retroactive since we didn't make the declaration during the release cycle, but two cycles (ish, I guess 1.23 hasn't started) after it was done